### ISSUE TITLE
Avatar should support separate button label and image alt text

### ISSUE THREAD

**Issue by mannycarrera4** [Workday/canvas-kit#77]
## 🐛 Bug Report

The Avatar component needs some documentation updates and we need to move `{...elemProps}` to the end so we can support overriding the aria label of the button.

## To Reproduce

Try to add `altText` and an `aria-label` to the avatar component. The altText prop will override the aria label, therefore missing the aria label. 

## Expected Behavior
The consumer should be able to pass an alt text for an image, and an aria label in the avatar component if it's a button.

Include a screenshot of a visual bug
![Screen Shot 2019-08-08 at 1 32 49 PM](https://user-images.githubusercontent.com/7966550/62732246-1f92b800-b9e1-11e9-83b9-17b608a96488.png)

**Comment by mannycarrera4** [comment]
This is being addressed by some further documentation improvements in another issue.
